src/PKSim.Core/Snapshots/Chart.cs

```diff
@@ -0,0 +1,18 @@
+﻿using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Snapshots
+{
+   public class Chart : IWithName, IWithDescription
+   {
+      public string Name { get; set; }
```

Comment:
Those properties are the same in Chart and Pop Analysis. I have a special mapper now for that, that only saves stuff when they have change in order to keep the json as simple as possible and not clutter it with internal stuff